python/activator/middleware_interface.py

```diff
@@ -105,6 +119,13 @@ def __init__(self, input_repo: str, image_bucket: str, instrument: str):
         self.calib_types = ["bias", "dark", "defects", "flat", "fringe", ]
 
     def prep_butler(self, visit: Visit) -> None:
+        """Prepare a temporary butler repo to process the incoming data in.
+
+        Parameters
+        ----------
+        visit : Visit
+            The visit to prepare the butler for.
```

Comment:
Note that a `Visit` object does not represent a visit, but (in Stack terms) an exposure. It is detector-specific, but not snap-specific.

Reply:
I don't think that's true? It has a `snaps` member, so it's a visit, not an exposure (assuming I'm understanding this correctly).

Reply:
There is a separate `Visit` object for each detector; see https://github.com/lsst-dm/prompt_prototype/blob/main/python/tester/upload.py#L65 for the constructor call.

Reply:
Oh, right! It's a "detector-visit". It's not an "exposure" in stack terms (or even an `afw.image.Exposure`), because that's a single snap, or a combined snap, whereas this is the snap-group. Which, come to think of it, is I think called a group? I should ask about this: a `Visit` is the combed object, but we're getting all the snaps here.

Oof, our terminology here is so confusing.

Reply:
Lets defer on any changes to `visit` for now. I think we might eventually want to name this a `DetectorVisit`, but see the new thred in #dm-naming-things: we don't really have a good term for this.

Reply:
I wasn't asking for changes to the `Visit` class, just that the documentation reflects that these are not full focal plane (which is what I assume when I hear "visit").